Core/include/Acts/Definitions/Units.hpp

```diff
@@ -195,4 +197,14 @@ ACTS_DEFINE_UNIT_LITERAL(mol)
 // not needed anymore. undef to prevent littering the namespace
 #undef ACTS_DEFINE_UNIT_LITERAL
 }  // namespace UnitLiterals
+
+/// Physical constants in native units.
+///
+/// Unit constants are intentionally not listed.
+namespace PhysicalConstants {
+/// Reduced Planck constant h/2*pi.
+inline constexpr double hbar =
+    6.582119569e-16 * UnitConstants::eV * UnitConstants::s;
```

Comment:
This is probably some extreme nitpicking but shouldn't this be 6.582119570e-16 ? I think the next digit is a 5 so we should round up (6.62607015/(2*1.602176634*3.141592653589793238)=0.6582119569509065) . Or maybe I am using some incorrect value in that case ignore that comment.

Reply:
I took the value listed on [Wikipedia](https://en.wikipedia.org/wiki/Planck_constant#Value). Maybe a rounding issue?

Reply:
I did the same but with the French page :) and when I saw a difference I redid the math with the pdg number. I Think this is because they write it : 6.582119569...e-16 which mean it is not rounded but that there are more number after the 9.

Reply:
The [Review of Particle Physics](https://pdg.lbl.gov/2020/reviews/rpp2020-rev-phys-constants.pdf) recommends the `...69` value.

Reply:
Interesting ! Lets go with that then (I looked at that page but completely missed that line...)

Reply:
The little star at the bottom confirm what I was thinking : 
∗These are calculated from exact values and are exact to the number of places given (i.e.no rounding). 
But lets not waste time on that !